crates/turbopack-ecmascript/src/references/mod.rs

```diff
@@ -262,27 +263,30 @@ pub(crate) async fn analyze_ecmascript_module(
                     }
                 }
             }
-            comments.trailing.iter().for_each(|r| {
-                r.value().iter().for_each(|comment| match comment.kind {
-                    CommentKind::Line => {
-                        lazy_static! {
-                            static ref SOURCE_MAP_FILE_REFERENCE: Regex =
-                                Regex::new(r#"# sourceMappingURL=(.*?\.map)$"#).unwrap();
-                        }
-                        if let Some(m) = SOURCE_MAP_FILE_REFERENCE.captures(&comment.text) {
-                            let path = &m[1];
-                            // TODO this probably needs to be a field in EcmascriptModuleAsset so it
-                            // knows to use that SourceMap when running code generation.
-                            // The reference is needed too for turbotrace
-                            analysis.add_reference(SourceMapReferenceVc::new(
-                                source.path(),
-                                source.path().parent().join(path),
-                            ))
+            if !is_inside_node_modules {
```

Comment:
Are we sure that we don't want source maps for `node_modules` packages? What about Next.js `transpilePackages`?

Reply:
I don't think we want to disable that at all in next.js. SourceMaps for everyone